src/media.html

```diff
@@ -111,1148 +111,1177 @@ <h1>Media</h1>
+              <span class="sr-only">Error:</span> Enter a valid email address
+            </div>
+          </div>
 
-  <div class="bs-callout bs-callout-accessibility" id="callout-glyphicons-accessibility">
-    <span class="accessibility-tag"><span class="icon accessibility"></span> Accessibility Callout</span>
-    <h4>Considerations when using icons</h4>
-    <p>
-      Modern versions of assistive technologies will announce CSS generated content, as well as specific Unicode characters. To avoid unintended and confusing output in screen readers (particularly when icons are used purely for decoration), we hide them with the <code>aria-hidden="true"</code> attribute.
-    </p>
-    <p>
-      If you're using an icon to convey meaning (rather than only as a decorative element), ensure that this meaning is also conveyed to assistive technologies – for instance, include additional content, visually hidden with the <code>.sr-only</code> class.
-    </p>
-    <p>
-      If you're creating controls with no other text (such as a <code>&lt;button&gt;</code> that only contains an icon), you should always provide alternative content to identify the purpose of the control, so that it will make sense to users of assistive technologies. In this case, you could add an <code>aria-label</code> attribute on the control itself.
-    </p>
-  </div>
-<figure class="highlight"><pre><code class="language-html" data-lang="html"><span class="nt">&lt;span</span> <span class="na">class=</span><span class="s">"icon search"</span> <span class="na">aria-hidden=</span><span class="s">"true"</span><span class="nt">&gt;&lt;/span&gt;</span></code></pre></figure>
+          <h2>Non-Functional Icons</h2>
+          <p>Non-Functional Icons come from the "Pastel" set of the Icons8 suite, and can be used for decorative purposes, such as those that reinforce information architecture. Non-Functional icons are not packaged in a typeface, but are implemented on an individual basis as SVGs.</p>
+          <div class="bs-example" data-example-id="non-functional-icon-examples">
+            <div class="ex-row img">
+              <div class="ex-col-4"><img src="assets/img/pastels/icons8-bell.svg"></div>
+              <div class="ex-col-4"><img src="assets/img/pastels/icons8-doctors_bag.svg"></div>
+              <div class="ex-col-4"><img src="assets/img/pastels/icons8-fingerprint_scan.svg"></div>
+              <div class="ex-col-4"><img src="assets/img/pastels/icons8-headset.svg"></div>
+            </div>
+            <div class="ex-row img">
+              <div class="ex-col-4"><img src="assets/img/pastels/icons8-money_box.svg"></div>
+              <div class="ex-col-4"><img src="assets/img/pastels/icons8-pills.svg"></div>
+              <div class="ex-col-4"><img src="assets/img/pastels/icons8-receipt_approved.svg"></div>
+              <div class="ex-col-4"><img src="assets/img/pastels/icons8-safety_float.svg"></div>
+            </div>
+          </div>
 
```

Comment:
Everything that follows is just code formatting/beautification, and should be able to be ignored.